DigitalLearningSolutions.Web/Views/LearningPortal/Shared/_TagRow.cshtml

```diff
@@ -48,5 +48,13 @@
       </div>
     }
   }
+  @if (Model is LearningResourceCardViewModel resource) {
+    <div class="tag nhsuk-u-font-size-14" aria-describedby="@Model.Id-name">
```

Comment:
I just gave these tags the same styling as the tags in the other Current cards for courses and self assessments. So these are consistent with the page, but not consistent with tags we use elsewhere (e.g. tracking system filters). 

Have a look at the screenshots and see if they look too much like buttons. Otherwise I can probably refactor the tags to look more like the tags we use elsewhere.

Reply:
I do think the tags look confusingly like buttons a bit too much, especially because they are the same color as the Mark as complete and Edit buttons. I don't know if "Image" and "WebLink" are test values or values likely to be displayed in the tags, but if they are, it seems likely users would click on them thinking that will open the image or link. I think it would be worth making they look like the tags we use elsewhere.

Reply:
I've updated the tags on the learning resource cards (and added new tags for each type). I've not updated the old course and self assessment tags yet. I will ask Kevin a question in the slack thread about the new tags to see what he thinks of them. Images below:

![Current page new tags desktop](https://user-images.githubusercontent.com/80777689/144420127-6c1ed271-aebf-475b-aac1-784e7aaa6e57.png)


<img src="https://user-images.githubusercontent.com/80777689/144420128-02ae5767-55d1-4f2e-a16e-a0907a428756.png" width=320px />

Reply:
I think that looks a lot better! And I support changing them on the other cards for consistency as long as Kevin okays that